deploy/charts/operator/values.yaml

```diff
@@ -42,13 +42,14 @@ operator:
   # -- Pod security context settings
   podSecurityContext:
     runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
 
   # -- Container security context settings for the operator
   containerSecurityContext:
     allowPrivilegeEscalation: false
     readOnlyRootFilesystem: true
     runAsNonRoot: true
-    runAsUser: 1000
```

Comment:
@ChrisJBurns would you prefer if this change was done in a separate `values-openshift.yaml` file?

Either way it looks like we need to bump the chart version (`deploy/charts/operator/Chart.yaml` and `deploy/charts/operator/README.md`)